opm/autodiff/BlackoilTransportModel.hpp

```diff
@@ -158,6 +158,13 @@ namespace Opm {
 
 
 
+        std::vector<V>
+        computeFluidInPlace(const ReservoirState& x,
+                            const WellState& xw,
+                            const std::vector<int>& fipnum)
+        {
+            return asImpl().computeFluidInPlace(x, xw, fipnum);
+        }
```

Comment:
flow_sequential complains segmentation fault here, and that's strange, because flow runs without problems.
But when I add any statement here like ``std::cout << "Debug" << std::endl``, flow_sequential runs without problems too, what's wrong here?